app/views/tour_list/recordings/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h3>Upload a Recording for <%= link_to @recording.landmark.name, landmark_path(@recording.landmark) %></h3>
```

Comment:
Again, is this copy/pasted from somewhere else? Bad habit -- what if we want to change something (I know Martin is planning to)?

Reply:
Yes, this is copy/pasted from somewhere else. No, this isn't the final code. 

This PR is to get the feature working and up so other team members can interact with it. I wanted to get feedback before I refactor anything so I don't waste my time on something that was going the wrong direction.

Reply:
RED, GREEN, REFACTOR